restic/__init__.py

```diff
@@ -34,6 +35,10 @@ def copy(*args, **kwargs):
     return internal_copy.run(_make_base_command(), *args, **kwargs)
 
 
+def find(*args, **kwargs):
```

Comment:
Since we're adding a new command, can we exercise `find()` in `e2e/test.py`? For example, try calling `find('mydata.txt')` and verify that it has a match, and try calling `find('non-existent-file.txt')` and verify that there's no match.